pyproject.toml

```diff
@@ -42,20 +42,20 @@ classifiers = [  #! Update me
 
 [tool.poetry.dependencies]
 python = "^3.8"
-pandas = "^1.3.0"
-httpx = "^0.23.0"
+pandas = ">=1.3.0"
+httpx = ">=0.23.0"
```

Comment:
upgraded to handle the conflicts with cohere==4.0 https://dominodatalab.atlassian.net/browse/DOM-56097